GitTrends/ViewModels/RepositoryViewModel.cs

```diff
@@ -118,7 +129,14 @@ public bool IsRefreshing
             get => _isRefreshing;
             set => SetProperty(ref _isRefreshing, value);
         }
+        string _totalButtonText = "TOTAL";
```

Comment:
Let's reference the RESX file, `RepositoryPageConstants.TOTAL`, instead of hard-coding the string